<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_window</class>
 <widget class="QWidget" name="main_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>932</width>
    <height>628</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pattern Tracker</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLineEdit" name="pathArea"/>
        </item>
        <item>
         <widget class="QPushButton" name="selectDirectoryButton">
          <property name="text">
           <string>Select directory</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="addToTrackingButton">
          <property name="text">
           <string>Add to tracking</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="notification">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="selectedDirectoriesView">
        <attribute name="horizontalHeaderCascadingSectionResizes">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>100</number>
        </attribute>
        <column>
         <property name="text">
          <string>Path</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Indexing</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Status</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="removeDirectoryButton">
          <property name="text">
           <string>Remove from list</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="stopIndexingButton">
          <property name="text">
           <string>Stop Indexing</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="continueIndexingButton">
          <property name="text">
           <string>Continue Indexing</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_2">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="searchStatus">
        <property name="text">
         <string>Enter the request</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLineEdit" name="requestArea"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QListWidget" name="foundPathesArea"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
